apollo-api/src/main/java/com/apollographql/android/api/graphql/Response.java

```diff
@@ -18,6 +21,10 @@ public boolean isSuccessful() {
     return errors == null || errors.isEmpty();
   }
 
+  @Nonnull public Operation getOperation() {
```

Comment:
nitpick - if there is no setOperation I don't think the `get` prefix is necessary